src/BenchmarkDotNet.Runtime/Portability/RuntimeInformation.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using BenchmarkDotNet.Environments;
+
+namespace BenchmarkDotNet.Portability
+{
+    public abstract class RuntimeInformation
+    {
+        internal const string Unknown = "?";
+        internal const string DebugConfigurationName = "DEBUG";
+        internal const string ReleaseConfigurationName = "RELEASE";
+
+        public virtual bool IsMono => false;
```

Comment:
Probably (I'm not sure), it would be better to use another `Runtime` enum here (which can be `Mono`, `Clr`, or `Core`).